api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreServiceImpl.java

```diff
@@ -470,6 +470,29 @@ public BiometricData saveBiometricsForPatient(Patient patient, BiometricData bio
 		return result;
 	}
 
+	@Override
+	public Patient findByPatientIdentifier(String patientIdentifier, String patientIdentifierTypeUuid) {
```

Comment:
What is the reason for placing this in the registration core service?

Reply:
I need to use it in both RegistrationApp and M2Sys. The better place for this would be PatientService, but it is the class from OpenMRS core, so I can't do this

Reply:
I see.

PatientService is ideal, but I will let this fly for now. We will need to refactor a few anyway to contribute to core.